docs/wiki/Home.md

```diff
@@ -14,20 +14,20 @@ The following instructions assume that you have already forked **dcrdex**.
 
 1. Initialize your forked wiki on github by navigating to the wiki tab of your forked **dcrdex** repo and clicking the "Create the first page" button. Be sure to "Save page" as well.
 
-<img src="images/wiki-creation.png" width="700">
+    <img src="images/wiki-creation.png" width="700">
 
 2. In your terminal, navigate to your local **dcrdex** directory and add your remote wiki.
 
-```
-git remote add wiki https://github.com/your-username/dcrdex.wiki.git
-git push wiki -d master
-git subtree push --prefix docs/wiki wiki master
-```
+    ```text
+    $ git remote add wiki https://github.com/your-username/dcrdex.wiki.git
+    $ git push wiki -d master
+    $ git subtree push --prefix docs/wiki wiki master
+    ```
 
-3. Now after committing changes to the files in the docs/wiki folder, you can apply them to your dcrdex wiki and view immediately. Assuming you are in the root directory:
+3. Now after committing changes to the files in the docs/wiki folder, you can apply them to your dcrdex wiki and view them immediately. Assuming you are in the root directory:
 
-```
-git push wiki $(git subtree split --prefix docs/wiki):master --force
-```
+    ```text
+    $ git push wiki $(git subtree split --prefix docs/wiki):master --force
```

Comment:
no `$` and use `bash` instead of `text.  There's lots of shell syntax here, namely the subshell.